src/starcheck.pl

```diff
@@ -578,6 +566,7 @@ sub json_obsids{
 
     $obs{$obsid}->check_monitor_commanding(\@bs, $or{$obsid});
     $obs{$obsid}->check_flick_pix_mon();
+    $obs{$obsid}->set_dynamic_mag_limits();
```

Comment:
Do we want a start date for the use of the dynamic mag limits in the code?  There could also be an option to turn off the dynamic limits... Mostly I'm wondering about the regression test set.